### ISSUE TITLE
Isolated environment. Possible incorrect projects for MSBuild Properties ~ $(TargetFrameworkVersion:Metadata)

### ISSUE THREAD

**Issue by 3F** [3F/vsSolutionBuildEvent#41]
Noticed from DllExport project:

```java
$(pMeta = "Metadata")
...
Metadata .NET: $(TargetFrameworkVersion:$(pMeta))
```

may return `Metadata .NET:  v4.0 ` ([1.6.0-beta2](https://www.nuget.org/packages/DllExport/1.6.0-beta2)) and `Metadata .NET:  v2.0` ([1.6.0-beta3](https://www.nuget.org/packages/DllExport/1.6.0-beta3))

When `<TargetFrameworkVersion>v2.0</TargetFrameworkVersion>` https://github.com/3F/DllExport/blob/20eb9db848f4ae4f730a311c1e22567b83ded629/Metadata/Metadata.csproj#L9

**Comment by 3F** [comment]
Seems problem only for Isolated env (CI, etc.), and probably because of this:
https://github.com/3F/vsSolutionBuildEvent/blob/e6f5f68fab83a145f71638e8600158996b3330a0/vsSolutionBuildEvent/IsolatedEnv.cs#L263

But anyway we need to replace this logic by new [MvsSln](https://github.com/3F/MvsSln) project.

**Comment by 3F** [comment]
Confirmed an fixed status after MvsSln integration #53

Verified via the same old changes:
https://github.com/3F/DllExport/releases/tag/v1.6-beta2
